contracts/DCAStrategies/DCAStrategies/DCAStrategiesPositionsHandler.sol

```diff
@@ -57,6 +49,11 @@ abstract contract DCAStrategiesPositionsHandler is IDCAStrategiesPositionsHandle
     return _positionId;
   }
 
+  /// @inheritdoc IDCAStrategiesPositionsHandler
+  function userPosition(uint256 _positionId) external view returns (Position memory _position) {
```

Comment:
Could you move this before `deposit`? We usually put view functions before non-view functions